styles/hackage/catppuccin.user.css

```diff
@@ -0,0 +1,554 @@
+
+    input#searchQuery ~ input[type="submit"]:hover {
+      background-color: @crust;
+    }
+
+    /*
+     * Others
+     */
+
+    .candidate-warn {
+      background-color: @mantle;
+      border-color: @red;
+    }
+
+    .candidate-info {
+      background-color: @mantle;
+      border-color: @blue;
+    }
+
+    .box {
+      background-color: @base;
+      border-color: @base;
+    }
+
+    div#modal #content,
+    #search-results {
+      background-color: @surface0;
+    }
+
+    .subs,
+    .top > .doc,
+    .subs > .doc {
+      border-left-color: @surface1;
+    }
+
+    .subs .subs p.src {
+      background-color: @mantle;
+    }
+
+    .top p.src {
+      border-color: @surface0;
+    }
+
+    .warning,
+    strong.warning {
+      color: @red;
+    }
+
+    small.info,
+    .flags-table .flag-disabled {
+      color: @subtext0; /* #888 */
+    }
+
+    .text-button {
+      color: @accentColor;
+    }
+
+    p.registration-email {
+      border-color: @accentColor;
+    }
+
+    :target {
+      background-image: linear-gradient(
+        to bottom,
+        transparent 0%,
+        transparent 65%,
+        mix(@base, @yellow, 40%) 60%,
+        mix(@base, @yellow, 40%) 100%
+      );
+    }
+
+    :target:hover {
+      background-image: linear-gradient(
+        to bottom,
+        transparent 0%,
+        transparent 0%,
+        mix(@base, @yellow, 40%) 0%,
+        mix(@base, @yellow, 40%) 100%
+      );
+    }
+
+    #footer {
+      background-color: @crust;
+      color: @subtext1;
+      border-color: @overlay0;
+    }
+
+    /* .methods, .constructors {
+      background-color: @base; border-color: @surface0;
+    } */
+
+    /* MathJax */
+    #MathJax_Message {
+      background-color: @mantle;
+      border-color: @surface1;
+      color: @text;
+    }
+
+    /*
+     * Sources
+     */
+
+    /* Old */
+
+    .hs-keyglyph,
+    .hs-layout {
+      color: @overlay2;
+    }
+    .hs-comment a {
+      color: @overlay0;
+    }
+    .hs-str,
+    .hs-chr {
+      color: @green;
+    }
+
+    /* Modern */
+
+    .hs-identifier {
+      color: @blue;
+    }
+    .hs-identifier.hs-type {
+      color: @yellow;
+    }
+    .hs-keyword {
+      color: @mauve;
+    }
+    .hs-string,
+    .hs-char {
+      color: @green;
+    }
+    .hs-number {
+      color: @peach;
+    }
+    .hs-operator {
+      color: @sky;
+    }
+    .hs-glyph,
+    .hs-special {
+      color: @overlay2;
+    }
+    .hs-comment {
+      color: @overlay0;
+    }
+    .hs-pragma {
+      color: @pink;
+    }
+    .hs-cpp {
+      color: @pink;
+    }
+
+    pre a:link,
+    pre a:visited {
+      border-bottom-color: @surface0;
+    }
+    pre a:hover,
+    pre a.hover-highlight {
+      background-color: @surface0;
+    }
+
+    span.annot {
+      color: @text;
+    }
+    span.annot:hover {
+      background-color: @surface1;
+    }
+    span.annot span.annottext {
+      background-color: @surface0;
+      border-color: @accentColor;
+    }
+  }
```

Comment:
Hmm, the rest of the userstyles use `accentColor` for the Stylus dropdown and then `accent-color` inside `#catppuccin { ... }`. Not really that important but it could be confusing having them both named the same thing.

```suggestion
    @accent-color: @catppuccin[@@lookup][@@accent];

    :root {
      color-scheme: if(@lookup = latte, light, dark);
    }

    html {
      background-color: @base;
    }

    body {
      background-color: @base;
      color: @text;
    }

    a:hover {
      background-color: inherit;
    }

    a[href]:link {
      color: @accent-color;
    }

    a[href]:visited {
      color: mix(@text, @accent-color, 75%);
    }

    a[href].def:link,
    a[href].def:visited {
      color: @subtext0;
    }

    a[href].def:hover {
      color: mix(@text, @subtext0, 40%);
    }

    .caption,
    h1,
    h2,
    h3,
    h4,
    h5,
    h6,
    summary {
      color: @accent-color;
      filter: none;
    }

    .collapser::before,
    .expander::before,
    .noexpander::before {
      color: @accent-color;
    }

    pre {
      background-color: @mantle;
      border-color: @surface1;
    }

    blockquote {
      background-color: @surface0;
      border-color: @accent-color;
    }

    .src {
      background-color: @mantle;
    }

    /*
     * Headers
     */

    #page-header,
    #package-header {
      background-color: @accent-color;
      color: @surface1;
      border-color: shade(@accent-color, 25%);
    }

    #page-header .caption,
    #package-header .caption,
    #page-header a:link,
    #page-header a:visited {
      color: @base !important;
    }

    #module-header .caption {
      color: @accentColor;
      border-color: @surface1;
    }

    #module-header {
      background-color: inherit;
    }

    #package-header #page-menu a:link,
    #package-header #page-menu a:visited,
    #package-header a:link,
    #package-header a:visited {
      color: @base;
    }

    #package-header a:hover {
      background-color: transparent;
    }

    /*
     * Header links
     */

    #package-header ul.links li {
      border-color: @overlay0;
    }

    #package-header ul.links li:hover {
      background-color: transparent;
    }

    ul.links li form input,
    ul.links li form button {
      color: @base;
    }

    /*
     * Style selectors (for old packages)
     */

    #style-menu {
      background-color: @mantle;
    }

    #style-menu li,
    #style-menu li a {
      color: @text !important;
    }

    #style-menu li + li {
      border-color: @overlay2;
    }

    /*
     * Versions
     */

    a.unpreferred[href]:link,
    a.unpreferred[href]:visited {
      color: @yellow;
    }

    a.deprecated[href]:link,
    a.deprecated[href]:visited {
      color: @red;
    }

    .versions a.normal[href]:link,
    .versions a.normal[href]:visited {
      color: @green;
    }

    /*
     * Readme style
     */

    .embedded-author-content {
      border-color: @accent-color;
    }

    /*
     * Contents & synopsis
     */

    #contents-list {
      background-color: @mantle;
    }

    #table-of-contents {
      background-color: @base;
      border-color: @surface1;
    }

    #synopsis summary {
      color: transparent;
    }

    #synopsis ul,
    #synopsis ul li.src {
      background-color: @surface0;
    }

    /*
     * Tables
     */

    table.info {
      background-color: @base;
      color: @subtext1;
      border-color: @surface1;
    }

    .doc th,
    .doc td table.fancy,
    table.fancy tr {
      border-color: @surface1;
    }

    .doc th,
    table.fancy tr.even td {
      background-color: @mantle;
    }

    table.fancy td,
    table.properties td,
    table.fancy th,
    table.properties th,
    table.dataTable.compact.fancy tbody th,
    table.dataTable.compact.fancy tbody td {
      background-color: @base;
    }

    table.fancy,
    table.fancy tr {
      border-color: @surface1;
    }

    div#table_length.dataTables_length,
    div#table_filter.dataTables_filter {
      color: @text;
    }

    div #description table,
    div #description table tr td {
      border-color: @surface1;
    }

    div #description table th {
      background-color: @mantle;
    }

    /*
     * Source links
     */

    #interface .src .selflink,
    #interface .src .link {
      color: @overlay2;
    }

    #interface span.fixity {
      color: @overlay1;
      border-left-color: @overlay1;
    }

    #interface span.rightedge {
      border-left-color: @overlay1;
    }

    #interface p.src .link {
      background-color: transparent;
      color: @overlay2;
      border-color: @overlay1;
    }

    /* Paginator */

    .paginator span {
      color: @text;
    }

    .paginator a {
      color: @overlay1;
    }

    .paginator a[href]:link,
    .paginator a:link:hover,
    .paginator a:visited:hover {
      color: @accent-color;
    }

    .paginator a:link:hover,
    .paginator a:visited:hover {
      background-image: none;
      background-color: @surface1;
    }

    .paginator .current,
    .paginator .current:hover {
      background-image: none;
      background-color: @surface0;
      color: @accent-color;
      border-color: @red;
    }

    .paginate_button {
      background-color: @overlay0;
    }

    /*
     * Search query
     */

    #browseTable th {
      border-color: @surface1;
    }

    #fatalError {
      color: @red;
    }

    input:invalid {
      border-color: @yellow;
      box-shadow: 0 0 10px @yellow;
    }

    input#searchQuery,
    input#searchQuery ~ input[type="submit"] {
      background-color: @mantle;
      border-style: solid;
      border-color: @surface0;
      border-radius: 4px;
    }

    input#searchQuery ~ input[type="submit"]:hover {
      background-color: @crust;
    }

    /*
     * Others
     */

    .candidate-warn {
      background-color: @mantle;
      border-color: @red;
    }

    .candidate-info {
      background-color: @mantle;
      border-color: @blue;
    }

    .box {
      background-color: @base;
      border-color: @base;
    }

    div#modal #content,
    #search-results {
      background-color: @surface0;
    }

    .subs,
    .top > .doc,
    .subs > .doc {
      border-left-color: @surface1;
    }

    .subs .subs p.src {
      background-color: @mantle;
    }

    .top p.src {
      border-color: @surface0;
    }

    .warning,
    strong.warning {
      color: @red;
    }

    small.info,
    .flags-table .flag-disabled {
      color: @subtext0; /* #888 */
    }

    .text-button {
      color: @accent-color;
    }

    p.registration-email {
      border-color: @accent-color;
    }

    :target {
      background-image: linear-gradient(
        to bottom,
        transparent 0%,
        transparent 65%,
        mix(@base, @yellow, 40%) 60%,
        mix(@base, @yellow, 40%) 100%
      );
    }

    :target:hover {
      background-image: linear-gradient(
        to bottom,
        transparent 0%,
        transparent 0%,
        mix(@base, @yellow, 40%) 0%,
        mix(@base, @yellow, 40%) 100%
      );
    }

    #footer {
      background-color: @crust;
      color: @subtext1;
      border-color: @overlay0;
    }

    /* .methods, .constructors {
      background-color: @base; border-color: @surface0;
    } */

    /* MathJax */
    #MathJax_Message {
      background-color: @mantle;
      border-color: @surface1;
      color: @text;
    }

    /*
     * Sources
     */

    /* Old */

    .hs-keyglyph,
    .hs-layout {
      color: @overlay2;
    }
    .hs-comment a {
      color: @overlay0;
    }
    .hs-str,
    .hs-chr {
      color: @green;
    }

    /* Modern */

    .hs-identifier {
      color: @blue;
    }
    .hs-identifier.hs-type {
      color: @yellow;
    }
    .hs-keyword {
      color: @mauve;
    }
    .hs-string,
    .hs-char {
      color: @green;
    }
    .hs-number {
      color: @peach;
    }
    .hs-operator {
      color: @sky;
    }
    .hs-glyph,
    .hs-special {
      color: @overlay2;
    }
    .hs-comment {
      color: @overlay0;
    }
    .hs-pragma {
      color: @pink;
    }
    .hs-cpp {
      color: @pink;
    }

    pre a:link,
    pre a:visited {
      border-bottom-color: @surface0;
    }
    pre a:hover,
    pre a.hover-highlight {
      background-color: @surface0;
    }

    span.annot {
      color: @text;
    }
    span.annot:hover {
      background-color: @surface1;
    }
    span.annot span.annottext {
      background-color: @surface0;
      border-color: @accent-color;
    }
  }
```